test/src/test_cases/edx-districts-frontend/test-district-user-school-access.js

```diff
@@ -0,0 +1,111 @@
+import {getToken} from "../../helpers/oauth-utils";
+import log from "npmlog";
+import {base_url, credentials} from "../../config/constants";
+import LoginPage from "../../page_models/login-page";
+import crypto from "crypto";
+import AccessUserCard from "../../page_models/access-user-card";
+import SnackBarPage from "../../page_models/common/snackBarPage";
+import HamburgerMenuPage from "../../page_models/common/hamburgerMenuPage";
+import InviteUserPage from '../../page_models/invite-user-page';
+import AccessUsersPage from '../../page_models/access-users-page';
+import InstituteSelectionPage from "../../page_models/institute-selection-page";
+
+const {deleteSetUpEdxUser,setUpEdxSchoolUserWithSpecificSchoolUserIdAndAllAvailableRoles,deleteSpecificEdxUser, setUpEdxDistrictUserWithAllAvailableRoles} =  require('../../helpers/user-set-up-utils');
+
+const loginPage = new LoginPage();
+const snackBarPage = new SnackBarPage();
+const menu = new HamburgerMenuPage();
+const newUserInvitePage = new InviteUserPage();
+const accessUsersPage = new AccessUsersPage();
+let createdEdxDistrictUser
+let accessUserCard = null;
+let token = '';
+
+fixture `district-users-access`
+    .before(async () => {
+        await setUpEdxDistrictUserWithAllAvailableRoles(['998'])
+        createdEdxDistrictUser = await setUpEdxSchoolUserWithSpecificSchoolUserIdAndAllAvailableRoles(crypto.randomUUID(), ['99998'])
+        accessUserCard = new AccessUserCard(createdEdxDistrictUser.edxUserID);
+        getToken().then(async (data) => {
+            token = data.access_token;
+            accessUserCard = new AccessUserCard(createdEdxDistrictUser.edxUserID);
+        }).catch((error => {
+            log.error("Failure during test setup: " + error);
+        }));
+    })
+    .after(async () => {
+        // find all test automation artifacts produced and remove them
+        log.info("Performing tear-down operation");
+        await deleteSpecificEdxUser(createdEdxDistrictUser.edxUserID);
+        await deleteSetUpEdxUser();
+    })
+    .beforeEach(async t => {
+        // log in as studentAdmin
+        await loginPage.login(credentials.adminCredentials);
+        await t.resizeWindow(1920, 1080);
+
+        //Select from hamburger menu
+        await menu.clickHamburgerMenu();
+        await menu.clickAdministrationMenuOption();
+        await menu.clickSchoolUserManagementSubMenuLink();
+        await t.wait(3000);
+    
+        //School User Management
+        await accessUserCard.verifySchoolSelectionCardExists();
+        await accessUserCard.verifySchoolSelectDropdownExists();
+        await accessUserCard.selectSchoolFromDropdown('EDX Automation Testing School');
+        await accessUserCard.clickManageSchoolButton();
```

Comment:
Just a style comment. Usually I skip the verify things exist. I figure if we can click the dropdown, we know it exists. Feel free to keep this in there since you did the work already. :) 

Testcafe has a default timeout of 3 seconds to find the element. https://testcafe.io/documentation/402638/reference/configuration-file#selectortimeout